server/opts.go

```diff
@@ -265,6 +265,10 @@ type WebsocketOpts struct {
 	// Users defined here or in the global options.
 	NoAuthUser string
 
+	// Name of the cookie, which if present, will be treated as JWT.
+	// If not valid or not present, protocol authorization is possible.
```

Comment:
This comment line is no longer true. Either remove or emphasize that if the client http request has a cookie of that name, it is treated as a JWT and needs to be valid for the connection to be authenticated.